README.rst

```diff
@@ -27,6 +27,7 @@ Requirements
 * Postgres 13
 * git
 * scancode-toolkit runtime dependencies (https://scancode-toolkit.readthedocs.io/en/stable/getting-started/install.html#install-prerequisites)
+* If you are using Ubuntu-20 or Ubuntu-22 LTS  ``sudo apt install libpq-dev``
```

Comment:
I think it would help to add some instruction for Fedora
```suggestion
* ``libpq-dev``
    * If you are using Debian or Ubuntu: ``sudo apt install libpq-dev``
    * If you are using Fedora: ``sudo dnf install libpq-devel``
```

Reply:
Hey @JonoYang can you review now :)

Reply:
Looks good!